client/src/Login.tsx

```diff
@@ -0,0 +1,165 @@
+import * as React from 'react';
+import { Link as RouterLink } from 'react-router-dom';
+import Paper from '@material-ui/core/Paper';
+import Typography from '@material-ui/core/Typography';
+import Grid from '@material-ui/core/Grid';
+import Button from '@material-ui/core/Button';
+import FormControl from '@material-ui/core/FormControl';
+import InputLabel from '@material-ui/core/InputLabel';
+import FilledInput from '@material-ui/core/FilledInput';
+import Link from '@material-ui/core/Link';
+import TextField from '@material-ui/core/TextField';
+import FacebookIcon from '@material-ui/icons/Facebook';
+import Visibility from '@material-ui/icons/Visibility';
+import VisibilityOff from '@material-ui/icons/VisibilityOff';
+import InputAdornment from '@material-ui/core/InputAdornment';
+import IconButton from '@material-ui/core/IconButton';
+import { makeStyles, Theme } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme: Theme) => {
+    const xPadding = 6;
+    const yPadding = 16;
+
+    return {
+        paper: {
+            width: 821,
+            height: 732,
+            borderRadion: 10,
+            paddingTop: theme.spacing(xPadding),
+            paddingBottom: theme.spacing(xPadding),
+            paddingLeft: theme.spacing(yPadding),
+            paddingRight: theme.spacing(yPadding)
+        },
+        button: {
+            borderRadius: 0,
+            height: 62
+        },
+        link: {
+            color: 'black'
+        }
+    };
+});
+
+function Login() {
+    const classes = useStyles();
+
+    interface UserLoginData {
+        email: string;
+        password: string;
+    }
+
+    const initialFormData: UserLoginData = {
+        email: '',
+        password: ''
+    };
+
+    const [ formData, setFormData ] = React.useState(initialFormData);
+    const [ showPassword, setShowPassword ] = React.useState(false);
+
+    const handleChange = (evt: React.ChangeEvent<HTMLInputElement>): void => {
+        const { name, value }: { name: string; value: string } = evt.target;
+        setFormData((fData) => ({
+            ...fData,
+            [name]: value
+        }));
+    };
+
+    // Toggle password visibility
+    const handleClickShowPassword = () => {
+        setShowPassword(!showPassword);
+    };
+
+    const handleMouseDownPassword = (evt: React.MouseEvent) => {
+        evt.preventDefault();
+    };
```

Comment:
is this really needed?

Reply:
No, it was in the Material-UI example when I brought it over. I've got it removed on my login-page branch. Maybe their example had the button as submit, and was submitting the form? Seems to work fine without it on my side. It'll be removed on the login-page PR